internal/daemon/controller/handlers/credentiallibraries/credentiallibrary_service_test.go

```diff
@@ -809,42 +809,6 @@ func TestCreate(t *testing.T) {
 				},
 			},
 		},
-		{
-			name: "Create a valid vault CredentialLibrary with no type set subtype",
```

Comment:
Removed this test since it seems like the current CLI behavior is to force the user to provide a subtype?